perf/perfmon.py

```diff
@@ -37,6 +37,7 @@ def setUp(self):
 
         deps = ["gcc", "make"]
         if dist.name in ['Ubuntu', 'debian']:
+            process.system('apt-cache search ncurses')
```

Comment:
wondering why apt search require for package install, maybe in your setup package list not refreshed

Reply:
apt-get update？useless.

Reply:
no, what I meant i did not face this issue in my setup so I think it can be setup issue so aksing please revisit your setup so code change not require

Reply:
You can run the test in the newly installed operating system. May trigger this issue.